tests/unit/specs/components/staking/DelegationsOverview.spec.js

```diff
@@ -0,0 +1,40 @@
+import { shallowMount, createLocalVue } from "@vue/test-utils"
+import DelegationsOverview from "staking/DelegationsOverview"
+import validators from "../../store/json/validators.js"
+import Vuex from "vuex"
+
+describe(`DelegationsOverview`, () => {
+  const localVue = createLocalVue()
+  localVue.use(Vuex)
+
+  let wrapper, $store
+
+  const getters = {
+    bondDenom: `stake`,
+    yourValidators: validators
+  }
+
+  beforeEach(() => {
+    $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      state: {
+        delegation: {
+          loaded: true
+        }
+      },
+      getters: JSON.parse(JSON.stringify(getters)) // clone so we don't overwrite by accident
```

Comment:
If we continue using this we should just write a helper `JSONClone()`.

Reply:
I just removed it for now.